docker/Dockerfile_local

```diff
@@ -0,0 +1,49 @@
+FROM abrt/retrace-server-image
+MAINTAINER mmarusak@redhat.com
+
+USER root
+
+# Copy sources to the docker image
+COPY . /retrace-server
+
+# From not on work from retrace-server directory
+WORKDIR '/retrace-server'
+
+# Change owner of /retrace-server, clean git and install dependences
+RUN chown -R --silent retrace /retrace-server && \
+    chmod -R --silent g=u /retrace-server && \
+    dnf -y install rpm-build sudo git &&  \
+    git clean -dfx && \
+    eval dnf -y --setopt=strict=0 --setopt=tsflags=nodocs install $(./autogen.sh sysdeps)
```

Comment:
add "dnf clean all" here?

Reply:
No need, as dnf is run twice and local builds are only for development purposes where speed of build is more important than the size of image